.github/workflows/windows.yml

```diff
@@ -184,10 +184,16 @@ jobs:
             ..\mroonga\
 
       # Artifact
+      - name: Compress the artifact
+        run: |
+          ls mariadb-${{ matrix.mariadb-version }}-${{ matrix.package-platform }}
```

Comment:
Can we remove this?

Reply:
Thank you for your comment.
I've removed it.